VersionControlUI/src/DiffManager.cpp

```diff
@@ -125,6 +125,86 @@ void DiffManager::clear()
 	}
 }
 
+void DiffManager::showNameChangeInformation(Visualization::ViewItem* currentViewItem, const DiffSetup& diffSetup)
+{
+	QString message = "";
+
+	for (auto entry : nameChanges_)
+	{
+		auto id = entry.second;
+
+		auto oldNode = const_cast<Model::Node*>(diffSetup.oldVersionManager_->nodeIdMap().node(id));
+		auto newNode = const_cast<Model::Node*>(diffSetup.newVersionManager_->nodeIdMap().node(id));
+
+		message += DiffComparisonPair::computeObjectPath(oldNode) +
```

Comment:
It would be nice if the message has a header of some sort:
The following objects have been renamed: